newIDE/app/src/GameDashboard/GamesList.js

```diff
@@ -146,13 +146,7 @@ const getDashboardItemsToDisplay = ({
   orderBy: GamesDashboardOrderBy,
 |}): ?Array<DashboardItem> => {
   if (!allDashboardItems) return null;
-  let itemsToDisplay: DashboardItem[] = allDashboardItems.filter(
-    item =>
-      // First, filter out unsaved games, unless they are the opened project.
-      !item.game ||
-      item.game.savedStatus !== 'draft' ||
-      (project && item.game.id === project.getProjectUuid())
-  );
+  let itemsToDisplay: DashboardItem[] = allDashboardItems;
```

Comment:
I don't know if this array should be copied with a destructuring operation because it can be sorted below, affecting both `itemsToDisplay` and `allDashboardItems` I think

Reply:
By the way, I see this line `itemsToDisplay = itemsToDisplay.sort(...)` but it can be just `itemsToDisplay.sort(...)`

Reply:
uh you're right, I'll fix that